# Request
int32 robot_id  # LEFT=0, RIGHT=1, DUAL=-1
float64[14] joint_positions  # 14 joint angles: [left_arm(7), right_arm(7)] in radians
bool move_mode  # ABS=0, INCR=1
float64 velocity  # Joint velocity (rad/s)
float64 acceleration  # Joint acceleration (rad/s^2)
bool is_block  # TRUE for blocking, FALSE for non-blocking
---
# Response
bool success
string message
